app/models/individual_event.rb

```diff
@@ -11,7 +11,10 @@ class IndividualEvent < ApplicationRecord
   belongs_to :office
 
   validates :user, :organization, :event_type, :office,
-    :description, :date, :duration, presence: true
+            :description, :date, presence: true
+  validates :duration, numericality: { only_integer: true,
+                                       greater_than: 0,
+                                       less_than_or_equal_to: 1440 }
```

Comment:
lets take advantage of rails here and make this more readable

```rb
less_than_or_equal_to: 24.hours
```

Reply:
I'm afraid this doesn't work as expected. `duration` on `IndividualEvent` is an integer representing minutes.

```
irb(main):018:0> 61 < 1.hour
=> true
```

Reply:
Okay